base/stream.jl

```diff
@@ -876,6 +894,14 @@ function stop_reading(stream::LibuvStream)
     end
 end
 
+function readbytes!(s::LibuvStream, b::AbstractArray{UInt8}, nb=length(b))
```

Comment:
It seems to me this should share code with `read!`; I recall a significant amount of work went into that. Not clear we even need both read and readbytes.